src/plugins/irc-events/topic.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(irc, network) {
 			type: Msg.Type.TOPIC_SET_BY,
 			mode: chan.getMode(data.nick),
 			nick: data.nick,
-			when: data.when,
+			when: new Date(data.when * 1000),
```

Comment:
Yep, I get that we need it in the handlebars parser, but do we need it here as well, when it gets sent?